packages/workflow/src/NodeHelpers.ts

```diff
@@ -1017,15 +1136,14 @@ export function addToIssuesIfMissing(foundIssues: INodeIssues, nodeProperties: I
  * @param {string} path The path to the properties
  * @returns
  */
-export function getParameterValueByPath(nodeValues: INodeParameters, parameterName: string, path: string) {
-	return get(
-		nodeValues,
-		path ? path + '.' + parameterName : parameterName,
-	);
+export function getParameterValueByPath(
+	nodeValues: INodeParameters,
+	parameterName: string,
+	path: string,
+): unknown {
+	return get(nodeValues, path ? path + '.' + parameterName : parameterName);
```

Comment:
have you considered prefer-template eslint rule https://eslint.org/docs/rules/prefer-template ? 
```suggestion
	return get(nodeValues, path ? `${path}.${parameterName}` : parameterName);
```
I usually find template literals a lot clearer and prettier